tests/common/aws_test_runner.c

```diff
@@ -41,6 +41,7 @@
 
 /* Application version info. */
 #include "aws_application_version.h"
+#include "../../libraries/abstractions/pkcs11/test/iot_test_pkcs11_globals.h"
```

Comment:
It doesn't seem ideal that this header is required here. Why does this need to be a different pattern from the other library test groups?

Reply:
The header is not required, it was removed in 7272131.